src/screens/DetailGroupImage/index.js

```diff
@@ -7,18 +7,18 @@ import {
   Text,
   View,
   ImageBackground,
-  StatusBar
+  StatusBar,
 } from 'react-native';
 
-import Icon from 'react-native-vector-icons/Octicons';
-import {useNavigation} from '@react-navigation/native';
 import MemoIc_arrow_back_white from '../../assets/arrow/Ic_arrow_back_white';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated. Ensure that the `colors` module is correctly exported and contains all the color properties used in this file.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->